src/contracts/mainnet/AaveV3EthLusdPayload.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Ethereum} from 'aave-address-book/AaveV3Ethereum.sol';
+import {IGenericV3ListingEngine, AaveV3ListingEthereum} from 'aave-helpers/v3-listing-engine/AaveV3ListingEthereum.sol';
+
+/**
+ * @title This proposal lists LUSD on Aave V3 Ethereum
+ * @author @marczeller - Aave-Chan Initiative
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0xda3519f11e2308239d5f179b2579fe921b2a421eb752aba959779ee9ecea0d69
+ * - Discussion: https://governance.aave.com/t/arc-add-lusd-to-ethereum-v3-market/11522
+ */
+contract AaveV3EthLUSDPayload is AaveV3ListingEthereum {
+  address constant LUSD = 0x5f98805A4E8be255a32880FDeC7F6728C6568bA0;
+  address constant LUSD_USD_FEED = 0x3E7d1eAB13ad0104d2750B8863b489D65364e32D;
```

Comment:
wrong feed i think `0x3D7aE7E594f2f2091Ad8798313450130d0Aba3a0` should be correct one?
https://docs.chain.link/data-feeds/price-feeds/addresses?network=ethereum

Reply:
fixed, thanks!